freedom/neural_nets/transformations.py

```diff
@@ -100,6 +100,7 @@ def call(self, hit, params):
         out = tf.stack([
                  delta,
                  dist,
+                 #rho ?,
```

Comment:
Why not?

Reply:
right. I added that comment to include it next time I train. But it is commented out because i was using the current model